test-data/unit/check-generics.test

```diff
@@ -766,7 +766,7 @@ if not isinstance(s, str):
 
 z = None # type: TNode # Same as TNode[Any]
 z.x
-z.foo() # E: Some element of union has no attribute "foo"
+z.foo() # Any simplyfies Union to Any
```

Comment:
@gvanrossum Yes, I remember our conversation on ``python/typing`` tracker. I just wanted to mark somehow this test, so that when it will start failing after ``Union[Any, ...]`` PR is merged it will be clear that nothing is special here. I have expanded this comment now to clarify.